### DISCUSSION TITLE
How are you all customizing the user data/table?

### DISCUSSION THREAD

**Discussion by znwhite** [nextauthjs/next-auth#1632] [Help]
I'm using a remote PostgreSQL database with next-auth. For the `users` table that it [has you create](https://next-auth.js.org/schemas/postgres), it has columns for `id`, `email`, `name`, `image` and a couple other things. This is awesome, but I also am going to need this table to have a `username` column, and initializie this column with a random value to start (which the user can then edit in their account settings, similar to twitter). 

How would you all make this happen? What's the least hacky way?

Thanks!

**Comment by StefanSelfTaught** [comment]
I'm using prisma and it's super easy to create new columns and tables.
